.github/workflows/site.yml

```diff
@@ -34,6 +36,49 @@ jobs:
             ${{ runner.os }}-pip-
       - name: Install dependencies
         run: python3 -m pip install -r ./site/requirements.txt
+      - name: Get collaborators
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          gh api graphql -f query='{
+            repository(owner: "substrait-io", name: "substrait") {
+              collaborators {
+                edges {
+                  permission
+                }
+                nodes {
+                  name
+                  company
+                }
+              }
+            }
+          }' | jq '[
+            [
+              [.data.repository.collaborators.edges[] | .permission],
+              [.data.repository.collaborators.nodes[] | .name],
+              [.data.repository.collaborators.nodes[] | .company]
+            ]
+            | transpose[]
+            | {
+              permission: .[0],
+              name: .[1],
+              company: .[2]
+            }
+          ]' > collaborators.json
+      - name: smc.yaml
+        run: |
+          jq 'map(select(.permission == "ADMIN"))
+            | map({
+                "Name":.name,
+                "Company":.company
+              })' collaborators.json | yq -P > site/data/smc.yaml
```

Comment:
I know github actions come with `jq` pre-installed -- is the same true for `yq`?  Or do we need to use https://github.com/frenck/action-setup-yq?

Reply:
It's pre-installed: https://github.com/actions/runner-images/blob/e0371b1f6ff0803f8f938779af7fe6e705e2b4eb/images/linux/Ubuntu2204-Readme.md?plain=1#L108